src/sections/newcontactuspage/haveAChat/HaveAChat.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import styles from "./HaveAChat.css";
+import { Button } from "@nextui-org/button";
+import Image from "next/image";
+import { ICONS } from "@/utils/icons";
+
+
+const HaveAChat = () => {
+  return (
+    <div className="Backgroundforchat">
+      <div >
+        <p className="firstparap">Let's Have a Chat!</p>
+      </div>
+      <div className="Columnforchat">
+        {/* left column starts */}
+
+        <div className="Leftforchat">
+          <p>Name</p>
+          <input id="input" type="text" />
+          <p>Email</p>
+          <input id="input" type="email" />
+          <p>Reasons of Interest</p>
+          <div>
+            <input type="radio" id="huey" name="drone" value="huey"/>
+            <label htmlFor="huey">Project</label>
+          </div>
+          <div>
+            <input type="radio" id="dewey" name="drone" value="dewey" />
+            <label htmlFor="dewey">Recruitment</label>
+          </div>
+          <div>
+            <input type="radio" id="louie" name="drone" value="louie" />
+            <label htmlFor="louie">Other</label>
+          </div>
+   
+          <div className="captcha">
+          <img src="/assets/Captcha.png"  alt="Captcha"/>
```

Comment:
**Suggestion**: If possible, use `<Image />` instead of `<img>`. *(Not Mandatory)*